test/test_manifest.json

```diff
@@ -6290,5 +6290,12 @@
       "rounds": 1,
       "link": true,
       "type": "other"
+   },
+   { "id": "bug1737260",
+      "file": "pdfs/bug1737260.pdf",
+      "md5": "8bd4f810d30972764b07ae141a4afbc4",
+      "rounds": 1,
+      "link": true,
+      "type": "eq"
```

Comment:
Shouldn't this also have `annotations: true` (see https://github.com/mozilla/pdf.js/blob/571fe0ead8cbdd26c00cc509ba40fe38c4f030bc/test/test_manifest.json#L6174 for example)?